appendices/common-problems/text.xml

```diff
@@ -161,7 +161,7 @@ directories Executable stacks ... binary files whose stack is marked with
 </section>
 
 <section>
-<title>Handling <c>repoman</c> Messages</title>
+<title>Handling <c>pkgcheck</c> Messages</title>
```

Comment:
```
<section>
<title>Handling <c>pkgcheck</c> Messages</title>
<body>

<todo>
write me
</todo>

</body>
</section>
```
Wanna go at it? ;)
Wonder if there should just be a link to pkgcheck documentation

Reply:
Actually maybe we want a whole new devmanual page dedicated to pkgcheck/pkgdev, I could add some parts from #235 there.

Reply:
I think that would be great, and I'd be happy to contribute to it, but I don't want to block this patch on it.

Reply:
I'm saying maybe this section can be deleted altogether now you're touching it, since "nothing existed here before".